connect/json/src/main/java/org/apache/kafka/connect/json/JsonConverter.java

```diff
@@ -293,6 +295,17 @@ public void configure(Map<String, ?> configs) {
 
         fromConnectSchemaCache = new SynchronizedCache<>(new LRUCache<>(config.schemaCacheSize()));
         toConnectSchemaCache = new SynchronizedCache<>(new LRUCache<>(config.schemaCacheSize()));
+                
+        try {
+            byte[] schemaContent = config.schemaFileContent();
+            if (schemaContent != null && schemaContent.length > 0) {
+                JsonNode schemaNode = deserializer.deserialize("", schemaContent);
+                this.schema = asConnectSchema(schemaNode);
+            }
+        } catch (SerializationException e) {
+            throw new DataException("Provided schema is invalid , please recheck the schema you have provided", e);
```

Comment:
Altering wording to try and be consistent with the style of error messages elsewhere in the class

```suggestion
            throw new DataException("Failed to parse schema in converter config due to serialization error: ", e);
```